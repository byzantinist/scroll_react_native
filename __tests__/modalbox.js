import 'react-native';
import React from 'react';
import ModalBox from '../components/modalbox';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <ModalBox/>
  );
  expect(tree).toMatchSnapshot();
});

