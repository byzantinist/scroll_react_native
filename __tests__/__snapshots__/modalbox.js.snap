// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "backgroundColor": "#36485f",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    collapsable={undefined}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "white",
        "color": "black",
        "height": 5000,
        "marginTop": 667,
        "padding": 20,
        "transform": Array [
          Object {
            "translateX": 0,
          },
          Object {
            "translateY": 0,
          },
        ],
        "width": 750,
      }
    }
  />
</View>
`;
